src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -48,39 +47,50 @@
               Grid.Row="1"
               Grid.Column="0">
             <Grid Grid.Row="0" Margin="4,7,4,4">
-                <graphControl:Grapher Name="GraphingControl" 
-                                  EquationsSource="{x:Bind ViewModel.Equations, Mode=OneWay}"
-                                  ForceProportionalAxes="True"
-                                  UseSystemFocusVisuals="True"
-                                  VariablesUpdated="GraphVariablesUpdated"
-                                  LostFocus="OnGraphLostFocus"
-                                  LosingFocus="OnLoosingFocus">
+                <graphControl:Grapher Name="GraphingControl"
+                                      ForceProportionalAxes="True"
+                                      LosingFocus="OnLoosingFocus"
+                                      LostFocus="OnGraphLostFocus"
+                                      UseSystemFocusVisuals="True"
+                                      VariablesUpdated="GraphVariablesUpdated">
                     <graphControl:Grapher.Background>
                         <SolidColorBrush Color="White"/>
                     </graphControl:Grapher.Background>
-                    <graphControl:Grapher.EquationTemplate>
-                        <DataTemplate x:DataType="vm:EquationViewModel">
-                            <graphControl:Equation Expression="{x:Bind Expression, Mode=OneWay}" LineColor="{x:Bind LineColor, Mode=OneWay}"/>
-                        </DataTemplate>
-                    </graphControl:Grapher.EquationTemplate>
-
                 </graphControl:Grapher>
 
-                <StackPanel Grid.Row="0" Orientation="Horizontal" HorizontalAlignment="Right" VerticalAlignment="Top" Margin="0,0,10,10">
-                    <Button x:Name="ActiveTracing" Click="OnActiveTracingClick" MinWidth="40" MinHeight="40" Margin="0,0,10,0">
+                <StackPanel Grid.Row="0"
+                            Margin="0,0,10,10"
```

Comment:
nit: spacing

Reply:
Not clear what spacing you are referring to here?

Reply:
It looks like it's just on GitHub. I missed the lack of + on the left.